doc/source/Container.md

```diff
@@ -0,0 +1,130 @@
+# Running watson-conversation-slots-intro in a Container on Bluemix with Kubernetes
+
+This directory allows you to deploy the `watson-conversation-slots-intro` application into a container running on Bluemix, using Kubernetes
+
+The commands below use environment variables for various specific names. Either run the following to export the ENV variables, or substitue your names in the commands or exports:
+```
+$ export CLUSTER_NAME=Watson
+$ export CONVERSATION_SERVICE=conversation-service-watson-pizzeria
+$ export CONFIG_MAP=watson-pizzeria-config
+$ export POD_NAME=watson-pizzeria-pod.yml
+$ export KUBE_SERVICE=pizza-bot
+$ export CONTAINER_ENV_VARIABLE=CONVERSATION_SERVICE_WATSON_PIZZERIA
+```
+
+
+# Steps
+
+## Create the Kubernetes cluster
+
+* Follow the instructions to [Create a Kubernetes Cluster](https://github.com/IBM/container-journey-template).
+```
+$ bx cs cluster-create --name $CLUSTER_NAME
```

Comment:
this command looks redundant if the container-journey-template is followed

Reply:
Yeah, maybe. But it links $CLUSTER_NAME, which is used later. I don't think there's much danger it's going to cause a problem.